connectivity/libraries/ppp/source/ppp_service.cpp

```diff
@@ -60,7 +60,7 @@ extern "C" { // "pppos.h" is missing extern C
 /* Timeout to wait for PPP connection to be terminated
  * (LCP Terminate-Request is answered with Terminate-Ack)
  */
-#define PPP_TERMINATION_TIMEOUT           30000
+#define PPP_TERMINATION_TIMEOUT 30s;
```

Comment:
This shouldn't work - you don't have a using directive (and it's got a stray semicolon). Not used?

Reply:
i changed it accordingly to your suggestions